### PR TITLE
porting ask mode bug fix 354500 in master

### PR DESCRIPTION
**Pull request by nipunja**

### CODE REVIEW

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 [hunk 1]
```diff
@@ -1,23 +1,19 @@
 param (
     [string]$environmentName,
-    [string]$resourceFilteringMethod,
-    [string]$machineFilter,
+    [string]$machineNames,
```
Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: Should you increase the patch version?

Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: Looks fine to me otherwise.

**Comment by bryanmacfarlane** [comment]
Accept or close

**Comment by kmkumaran** [comment]
Old PR. Closing it.

**Pull request closed without merging**